public/index.html

```diff
@@ -7,6 +7,50 @@
   </head>
 
   <body>
+    <div id="main"> 
+      <p id="title">Dr. Norman Borlaug<br/><br/></p>
+      <p id="description">The man who saved a billion lives</p>
 
+      <div id="img-div">
+          <img id="image" src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg" alt="norman and collegues">
+          <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase 
+            wheat yields - part of his life-long war on hunger.</p>      
+      </div>
```

Comment:
- [ ] To best follow semantic practices, kindly change the wrapping `div` tag to a `section` tag and wrap the content within, it will enhance the readability of your code like in other places mentioned.